bin/layers.go

```diff
@@ -4,24 +4,15 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/containers/image/v5/image"
-	"github.com/containers/image/v5/transports/alltransports"
 	"github.com/containers/image/v5/types"
 	"github.com/urfave/cli/v3"
+
+	skiff "github.com/dcermak/skiff/pkg"
 )
 
 func ShowLayerUsage(url string, ctx context.Context, sysCtx *types.SystemContext) (string, error) {
-	ref, err := alltransports.ParseImageName(url)
-	if err != nil {
-		return "", err
-	}
-	imgSrc, err := ref.NewImageSource(ctx, sysCtx)
-	if err != nil {
-		return "", err
-	}
-	defer imgSrc.Close()
 
-	img, err := image.FromUnparsedImage(ctx, sysCtx, image.UnparsedInstance(imgSrc, nil))
+	img, err := skiff.ImageFromUrl(url, ctx, sysCtx)
```

Comment:
nit: given we use c/image and that supports the standard `transport:details` syntax:
```suggestion
	img, err := skiff.ImageFromReference(url, ctx, sysCtx)
```

Reply:
Even URI seems a bit weird to refer to "transport:details" but the preferred usage across c/* is to refer to the argument simply as "image".

URL sounds as if it _is_ a registry URL or maybe it's just me.